File: mesi.clipboard/src/main/kotlin/mesi/io/clipboard/data/ClipboardEntry.kt
package mesi.io.clipboard.data

import org.springframework.data.jpa.repository.JpaRepository
import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "clipboardentries")
data class ClipboardEntry (

        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id : Long,

        @Column(nullable = false, length = 512)
        val content : String,

        @Column(nullable = false)
        val timeStamp : LocalDateTime
)

interface ClipboardEntryRepo : JpaRepository<ClipboardEntry, Long>

File: src/Mesi.Io.Api.Clipboard/Models/ClipboardEntry.cs
using System;

namespace Mesi.Io.Api.Clipboard.Models
{
    public class ClipboardEntry
    {
        public ClipboardEntry(Guid userId, string content, DateTime createdAt)
        {
            UserId = userId;
            Content = content;
            CreatedAt = createdAt;
        }

        public Guid UserId { get; }
        public string Content { get; }
        public DateTime CreatedAt { get; }
    }
}